integration/update_field_test.go

```diff
@@ -86,7 +86,7 @@ func TestUpdateFieldCurrentDate(t *testing.T) {
 
 	t.Run("currentDate", func(t *testing.T) {
 		// maxDifference is a maximum amount of seconds can differ the value in placeholder from actual value
-		maxDifference := time.Duration(3 * time.Minute)
+		maxDifference := time.Duration(4 * time.Minute)
```

Comment:
This place started failing too often, and 3 min didn't seem to be enough anymore. 
We have some tasks that hopefully will improve performance, but for now it's out of the scope.

Reply:
Please add TODO with that task

Reply:
Added